### DISCUSSION TITLE
Is this only dev related?  Could this page be used for ideas outside of computing?

### DISCUSSION THREAD

**Discussion by wnadim92** [open-source-ideas/ideas#326] [General]
Could this apply for general open source projects?  I wanted to make a website called opensourceideas where people can do exactly that surprisingly what this github page does!

So I am asking if the purpose of this github page is just that, meaning it spans the scope outside of computer dev work.  

For example I have one myself of shock absorbing infrastructure designs which aims to provide collaboration from various people in varying expertise levels, to simply come to an earth quake/ environmental resistant infrastructure design.  Although it's not solveable as it can always be improved, it would bring us as a species having more open source ideas for earth quake resistance.

I myself am an Azure Solutions Architect Expert and realize in my career in IT as a whole, the importance of open source linux, and how it truly shaped the improvement of computers, with the community contributing to it.

I wanted to make a website that does kind of that but span outside of computing.  It would definitely be open to including computing, but really opensourceideas in all aspects :)

**Comment by wnadim92** [comment]
So I am asking if this github page already does that.  And if it does, I then will start posting a thread on earth quake resistant designs :)

**Comment by FredrikAugust** [comment]
Hey! While it's mainly focused on software development projects, I won't stop you from creating an issue for something not IT-related!
